messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "https://i.imgur.com/nly0Cmm.gifv",
+        "https://i.redd.it/dstee514i5zz.jpg",
+        "https://i.imgur.com/jWFy9aV.gifv",
+        "https://i.imgur.com/aK34qpZ.gifv"
+    ],
+    "latency": "{0:.3f} Sekunde(en).",
+    "lynch_reveal": [
+        "Die Dorfbewohner entscheiden nach vielen Debatten endlich über das Lynchen von {0:@} und {0:@} entpuppte sich als {1!role:article} {1!role:bold}.",
+        "Eine Abstimmung ist getroffen wurden, und die Dorfbewohner lynchten {0:@}, {1!role:article} {1!role:bold}.",
+        "Nach langem Ringen wird  {0:@} zum Galgen gebracht und nach dem Tod wurde entdeckt das {0:@} {1!role:article} {1!role:bold} war.",
+        "Die Dorfbewohner entscheiden sich {0:@} zu hängenfffff das Seil dehnt sich und reist, und der folgende Sturz tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner haben es schwer mit dem Schmerz des Todes von {0:@} und lynchten wiederwillig {1!role:article} {1!role:bold}.",
+        "Konform mit dem Willen des Dorfes, erwiesen sich die Galgen als wirksam bei der Tötung von {0:@}, {1!role:article} {1!role:bold}.",
+        "In einem Anfall von Hysterie lynchen die Dorfbewohner {0:@}, und töten {1!role:article} {1!role:bold}.",
+        "Während sie auf ihre Hinrichtung warten, fällt ein Baum auf {0:@}, dies tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner fesseln {0:@}. Und werfen {0:@} in einen See. Es scheint so das {1!role:article} {1!role:bold} nicht gut schwimmen kann",
+        "Glaubt man ihrem Nachbarn und Freund das {0:f@} gefährlich zu sein scheint, so legt die Masse der Dorfbewohner den {1!role:bold} {0:@} um.",
+        "Unter viel Lärm lynchen die Pechgabel-lagerischen Dorfbewohner {0:@}, was sich als... {1!role:article} {1!role:bold} entpuppte.",
+        "Trotz Protesten schleppt der Mob sein Opfer an den hängenden Baum. {0:@} erliegt dem Willen der Horde und wird gehängt. Die Dorfbewohner haben {1!role:article} {1!role:bold} getötet .",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Sobald das Zucken aufhörte, wurde entdeckt, dass das Dorf {1!role:article} {1!role:bold} gelyncht hat.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Danach inspiziert das Dorf die Leiche und findet {1!role:article} {1!role:bold}.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Leider reist das Seil, und {1!role:article} {1!role:bold} schlägt auf den Boden auf und stirbt.",
+        "Bevor das Seil gezogen wird, {0:@}, {1!role:article} {1!role:bold}, zieht den Stift an einer Granate. Sie zögert, und explodiert und tötet {0:@}.",
+        "Bevor das Seil gezogen wird wirft {0:@} {1!role:article} {1!role:bold}, eine Granate auf Menge doch die Granate explodierte zu früh.",
+        "Die Dorfbewohner ziehen {0:@} an den Rand einer Klippe. Bevor sie protestieren können, {1!role:article} {1!role:bold} rutscht und fällt in den Abgrund.",
```

Comment:
Bevor (player) protestieren kann, rutscht (role) und fällt in den Abgrund. (grammar, mistranslation of singular they)